### ISSUE TITLE
How to get a confidence of a specific class

### ISSUE THREAD

**Issue by yuhaoooo** [deepinsight/insightface#1152]
Thanks deepinsight team for the great work! I have a problem about finetuning model. 

The ArcLoss introduces a margin to help the embedding model to perform better. However, I already have a pretrained embedding model, and a tiny dataset.  So I should only train the ArcLoss head (except for the embedding model) to recognition the identities of this tiny dataset. In my opinion, I should set `m = 0` so that the output of ArcLoss head is confidence of each identity.

But after I train the arcloss head in this way, the predicted result of arcloss head is much more unstable compared to facenet (based on softmax) and cosface.

Do I have something wrong? And how do I use arcloss to predict the confidence? Thanks a lot.

**Comment by nttstar** [comment]
Please train a MLP network after getting the embeddings of your dataset.

**Comment by yuhaoooo** [comment]
Thanks a lot!
